packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -0,0 +1,66 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    borderWidth,
+    controlHeight,
+    fillHoverColor,
+    standardPadding,
+    tableRowBorderColor
+} from '../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('flex')}
+
+    :host {
+        align-items: center;
+        height: calc(${controlHeight} + 2 * ${borderWidth});
+        border-top: calc(2 * ${borderWidth}) solid ${tableRowBorderColor};
+        padding-left: calc(
+            ${standardPadding} * 2 *
+                var(--ni-private-table-group-row-indent-level)
+        );
+    }
+
+    :host(:hover) {
+        background: ${fillHoverColor};
+    }
+
+    :host([expanded='true']) .expander-icon,
+    :host([expanded]) .expander-icon {
+        transform: rotate(90deg);
+    }
+
+    :host([expanded='false']) .expander-icon {
+        transform: rotate(0deg);
+    }
+
+    :host([expanded='true']) .animating,
+    :host([expanded='false']) .animating {
+        transition: 0.2s ease-in-out;
+    }
+
+    .group-row-header-content {
+        display: flex;
+        overflow: hidden;
+    }
+
+    .group-header-value {
+        padding-left: calc(${standardPadding} / 2);
+        user-select: none;
+        overflow: hidden;
+        display: flex;
+    }
+
+    .group-row-child-count {
+        padding-left: 2px;
+        pointer-events: none;
+        user-select: none;
+    }
+
+    @media (prefers-reduced-motion) {
+        :host([expanded='true']) .animating,
+        :host([expanded='false']) .animating {
+            transition-duration: 0s;
```

Comment:
Does this cause problems with the `transitionend` event handling? I can't remember from my work on the slideout whether or not `transitionend` is called for animations where the duration is 0.

Reply:
I'll at least say that behaviorally, this seems to work as expected. When I turn off animations at the system level, I just see the arrow reverting to each rotation state immediately.

Reply:
I've verified that the `removeEventListener` callback is called on every click.